src/actions.cpp

```diff
@@ -397,7 +399,8 @@ static void showUseHotkeyMessage(Player* player, const Item* item, uint32_t coun
 	} else if (count == 1) {
 		player->sendTextMessage(MESSAGE_HOTKEY_PRESSED, fmt::format("Using the last {:s}...", item->getName()));
 	} else {
-		player->sendTextMessage(MESSAGE_HOTKEY_PRESSED, fmt::format("Using one of {:d} {:s}...", count, item->getPluralName()));
+		player->sendTextMessage(MESSAGE_HOTKEY_PRESSED,
```

Comment:
What about making `sendTextMessage` forward it's arguments to `fmt::format`? It would keep working for non-formatted strings because `fmt::format(string)` just returns the parameter unmodified.